src/client_side_request.cc

```diff
@@ -1115,7 +1115,7 @@ clientInterpretRequestHeaders(ClientHttpRequest * http)
         }
 
 #if USE_FORW_VIA_DB
-        fvdbCountVia(s.termedBuf());
+        fvdbCountVia(s);
```

Comment:
```suggestion
        fvdbCountVia(StringToSBuf(s));
```

Reply:
s is a SquidString. We can convert in client_side_request.cc or in access_log.cc, but somewhere we must.

I take from your comments that you prefer to do it in client_side_request.cc, so I'll do that

Reply:
Done in commit f927fe288